gunicorn/arbiter.py

```diff
@@ -558,6 +558,13 @@ def manage_workers(self):
                                   "value": active_worker_count,
                                   "mtype": "gauge"})
 
+        backlog = sum([sock.get_backlog() for sock in self.LISTENERS])
+        if backlog is not None:
```

Comment:
should this be `if backlog`? backlog will never be None right?

Reply:
`get_backlog()` can return None if the platform doesn't support it. I've updated the code to handle that scenario.